Chocolatey.Docs.Cake.Recipe/Content/parameters.cake

```diff
@@ -26,7 +27,8 @@ public static class BuildParameters
         context.Information("PublishDirectory: {0}", PublishDirectory);
         context.Information("OutputDirectory: {0}", OutputDirectory);
         context.Information("VirtualDirectory: {0}", VirtualDirectory);
-        context.Information("Target: {0}", Target);
+        context.Information("Port: {0}", Port);
+        context.Information("Target: {1}", Target);
```

Comment:
@gep13 I'm unsure if this should be changed to `1` or left as `0`, based on the changes to the `build.cake` file.

Reply:
This one should be `{0}`.

The only case you need to worry about anything more than `{0}` in one of these is when you're putting multiple values into a single string, like you are in the build.cake file. These are all emitted/formatting as separate strings, so they can all use `{0}`.